client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
+            <div className="name-img-card">
+              <div className="img-card">
+                <img src={imgUrl} alt="card" />
+              </div>
+              <div>
+                <h3>Name</h3>
+                <p>{name}</p>
+              </div>
+            </div>
+            {description && (
+              <div>
+                <h3>Description</h3>
+                <p>{description}</p>
+              </div>
+            )}
+            <div>
+              <h3>Github link</h3>
+              <NavLink exact to={githbUrl}>
+                Click
+              </NavLink>
+            </div>
+            {student && (
+              <div>
+                <h3>Student</h3>
+                <NavLink exact to="/admin/students">
+                  View
+                </NavLink>
+              </div>
+            )}
+            {websiteLink && (
+              <div>
+                <h3>Website</h3>
+                <NavLink exact to={websiteLink}>
+                  View
+                </NavLink>
+              </div>
+            )}
+            {studentNames && (
+              <div>
+                <h3>Student</h3>
+                <p>{studentNames}</p>
+              </div>
+            )}
+            {ComProject && (
+              <div>
+                <h3>Community p</h3>
+                <NavLink exact to="/admin/projects?type=internal">
+                  View
+                </NavLink>
+              </div>
+            )}
+            {CliProject && (
+              <div>
+                <h3>Clients p</h3>
+                <NavLink exact to="/admin/projects?type=remotely">
+                  View
+                </NavLink>
+              </div>
+            )}
+            <div>
+              <Button onClick={editCard} className="card-btn edit">
+                Edit
+              </Button>
+              <Button onClick={deleteCard} className="card-btn">
+                Delete
+              </Button>
+            </div>
+          </li>
+        </ul>
+      </section>
+      <section className="pagination">
+        <Pagination defaultCurrent={1} total={50} />
```

Comment:
I think it's not correct to put Pagination here for two reasons,  the first we will use this component more than one time on the page, 
each card will create Pagination, so will be a lot of Pagination on the same page, may you don't care about it because you make its position absolute and will appear one Pagination
![807](https://user-images.githubusercontent.com/55782435/78896690-86d5a000-7a79-11ea-928e-4cf216ce02d7.png)

 but actually it like this 
![806](https://user-images.githubusercontent.com/55782435/78896612-66a5e100-7a79-11ea-910a-a8449a096bf8.png)
the second we will need to use **state** to make Pagination work correctly

Reply:
I agree.
Once you remove the pagination from this component and move it into AdminCohortPage, you could then remove the fragment tag at the top (`<>` on line 21) and just have the section tag be the top-level tag for this component.

Reply:
I have resolved that. Thank you 😊